exla/Makefile

```diff
@@ -66,7 +66,7 @@ HEADERS = $(EXLA_DIR)/mlir/ops.h $(EXLA_DIR)/mlir/builder.h $(EXLA_DIR)/mlir/cus
 OBJECTS = $(patsubst $(EXLA_DIR)/%.cc,$(EXLA_CACHE_OBJ_DIR)/%.o,$(SOURCES)) $(EXLA_CACHE_OBJ_DIR)/exla_cuda.o
 
 
-NVCC_RESULT := $(shell which nvcc 2> NULL)
```

Comment:
This creates a NULL file for me :D

Reply:
ah! this is where that was coming from haha!